browser/ts/index.ts

```diff
@@ -842,6 +844,8 @@ function signedOrderToWrapperSignedOrder(signedOrder: SignedOrder): WrapperSigne
 function wrapperOrderEventToOrderEvent(wrapperOrderEvent: WrapperOrderEvent): OrderEvent {
     return {
         ...wrapperOrderEvent,
+        // tslint:disable-next-line:custom-no-magic-numbers
+        timestamp: Math.round(new Date(wrapperOrderEvent.timestamp).getTime() / 1000),
```

Comment:
I don't think we should reduce the precision here by dividing by 1000 and rounding. It seems like the extra precision could be useful.

How about either leaving the date as a string or converting it to milliseconds since the Unix Epoch instead of seconds? If we do go with milliseconds I would also suggest changing the name to something like `timestampMilliseconds` to reduce ambiguity.